src/Api/Groups/Group.php

```diff
@@ -19,6 +19,7 @@
 use Serhiy\Pushover\Client\Request\Request;
 use Serhiy\Pushover\Client\Response\AddUserToGroupResponse;
 use Serhiy\Pushover\Client\Response\CreateGroupResponse;
+use Serhiy\Pushover\Client\Response\ListGroupsResponse;
```

Comment:
@Vitexus lets push some code into this PR so that we get new cs-fixer and phpstan checks. For example, this line can be moved down so that imports are sorted.